nexuse2e-webapp/src/main/frontend/src/assets/i18n/en.json

```diff
@@ -0,0 +1,20 @@
+{
+  "login": {
+    "httpWarning": "You are trying to log in using an insecure connection. Using HTTPS is strongly advised.",
+    "wrongCredentials": "Wrong login data. Please try again."
```

Comment:
Should be more clear about this can be a wrong username or wrong password?

Reply:
Yes, I had something like "Wrong username or password. Please try again." in mind

Reply:
Now I get it ;) Changed the string accordingly